app/helpers/application_helper.rb

```diff
@@ -10,4 +10,8 @@ def resource
   def devise_mapping
     @devise_mapping ||= Devise.mappings[:user]
   end
+
+  def user_admin?
+    current_user.role.eql? "admin"
```

Comment:
should be ```current_user.admin?``` and what if current_user.nil?

Reply:
should be ```current_user.admin?``` and what if current_user.nil?